bot/exts/evergreen/tic_tac_toe.py

```diff
@@ -0,0 +1,325 @@
+        )
+    return check(predicate)
+
+
+class TicTacToe(Cog):
+    """TicTacToe cog contains tic-tac-toe game commands."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.games: t.List[Game] = []
+
+    @guild_only()
+    @is_channel_free()
+    @is_requester_free()
+    @group(name="tictactoe", aliases=("ttt",), invoke_without_command=True)
+    async def tic_tac_toe(self, ctx: Context, opponent: t.Optional[discord.User]) -> None:
+        """Tic Tac Toe game. Play against friends or AI. Use reactions to add your mark to field."""
+        if opponent == ctx.author:
+            await ctx.send("You can't play against yourself.")
+            return
+        if opponent is not None and not all(
+            opponent not in (player.user for player in g.players) for g in ctx.cog.games if not g.over
+        ):
+            await ctx.send("Opponent is already in game.")
+            return
+        if opponent is None:
+            game = Game(
+                [Player(ctx.author, ctx, Emojis.x), AI(Emojis.o)],
+                ctx
+            )
+        else:
+            game = Game(
+                [Player(ctx.author, ctx, Emojis.x), Player(opponent, ctx, Emojis.o)],
+                ctx
+            )
+        self.games.append(game)
+        if opponent is not None:
+            confirmed, msg = await game.get_confirmation()
+
+            if not confirmed:
+                if msg:
+                    await ctx.send(msg)
+                return
+        await game.play()
+
+    @tic_tac_toe.group(name="history", aliases=("log",), invoke_without_command=True)
+    async def tic_tac_toe_logs(self, ctx: Context) -> None:
+        """Show most recent tic-tac-toe games."""
+        if len(self.games) < 1:
+            await ctx.send("No recent games.")
+            return
+        log_games = []
+        for i, game in enumerate(self.games):
+            if game.over and not game.canceled:
+                if game.draw:
+                    log_games.append(
+                        f"**#{i+1}**: {game.players[0]} vs {game.players[1]} (draw)"
+                    )
+                else:
+                    log_games.append(
+                        f"**#{i+1}**: {game.winner} :trophy: vs {game.loser}"
+                    )
+        await LinePaginator.paginate(
+            log_games,
+            ctx,
+            discord.Embed(title="Most recent Tic Tac Toe games")
+        )
+
+    @tic_tac_toe_logs.command(name="show", aliases=("s",))
+    async def show_tic_tac_toe_board(self, ctx: Context, game_id: int) -> None:
+        """View game board by ID (ID is possible to get by `.tictactoe history`)."""
+        if len(self.games) < game_id:
+            await ctx.send("Game don't exist.")
+            return
+        game = self.games[game_id - 1]
+        await ctx.send(
+            f"{game.winner} :trophy: vs {game.loser}"
+        )
```

Comment:
I don't think this line should be multiple when every other line in this file has `await ctx.send()` as one line.
```suggestion
       await ctx.send(f"{game.winner} :trophy: vs {game.loser}")
```